sdk/synapse/synapse-access-control/review/synapse-access-control.api.md

```diff
@@ -4,157 +4,208 @@
 
 ```ts
 
-import * as coreHttp from '@azure/core-http';
-import { PagedAsyncIterableIterator } from '@azure/core-paging';
```

Comment:
this does not feel right

Reply:
Yeah, the Swagger changed the definition of this function and removed the pagination. Talked to @xiangyan99 and this seems to be a conscious change looking for feedback. Same change accross languages